descriptor/qsaratomic/src/test/java/org/openscience/cdk/qsar/descriptors/atomic/PartialTChargeMMFF94DescriptorTest.java

```diff
@@ -18,82 +18,23 @@
  */
 package org.openscience.cdk.qsar.descriptors.atomic;
 
-import org.junit.Assert;
 import org.junit.Before;
-import org.junit.Ignore;
-import org.junit.Test;
```

Comment:
Parent tests are run for the descriptors right? The before sets the class of descriptor. I tried to adapt these tests but since MMFF varies with tautomers it was impossible to match up the expected outputs. All have been migrated though to MmffTest.